packages/wonder-blocks-floating/src/components/floating.tsx

```diff
@@ -141,7 +140,6 @@ export default function Floating({
             // Shift along the axis to keep it in view
             shiftProp
                 ? shift({
-                      padding: boundaryPadding,
```

Comment:
I think we want to include the boundary padding anyways so the Floating element doesn't get too close to the boundaries, but the value doesn't need to be configurable! 

Related thread where we added boundary spacing for tooltips: https://khanacademy.slack.com/archives/C8Z9DSKC7/p1724766817685709 

(Turns out it was me that made the current `viewportPadding` prop configurable 😅 There are some good points in the thread about being able to configure where the arrow is though!)

Reply:
oh yeah, I know remember that. So, I checked in the codebase and we don't override that prop, so I'm going to hard-code a value (the same one we currently have in `viewportPadding`) but keep that padding internal for now. wdyt?

Reply:
Sounds good to me!